src/styles/styles.js

```diff
@@ -705,18 +633,16 @@ const styles = {
         alignItems: 'center',
         display: 'flex',
         flexDirection: 'row',
-        flexWrap: 'wrap',
     },
 
     chatItemMessageHeaderSender: {
         color: themeColors.heading,
-        fontFamily: fontFamily.GTA_BOLD,
         fontSize: variables.fontSizeNormal,
-        fontWeight: fontWeightBold,
+        height: 24,
         lineHeight: 20,
+        fontWeight: '600',
```

Comment:
The change to `fontWeight` here shouldn't be necessary - please use `fontWeightBold` instead